lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -184,6 +184,41 @@ function gutenberg_get_global_settings( $path = array(), $context = array() ) {
 }
 
 /**
+ * Repeated logic from `get_default_block_editor_settings` function. When implemented into core,
+ * remove logic from `get_default_block_editor_settings` and simple call this function instead.
+ *
+ * @return array
+ */
+function gutenberg_get_block_theme_supports() {
```

Comment:
Would you consider a name such as `gutenberg_get_legacy_theme_supports_for_theme_json`? Or something along that vein. It's more in line with what this code is used for. The current naming may imply that it returns all theme supports, and may mislead people.

Reply:
Orignally was in experimental, but it would mean, that this code would do nothing in the plugin - https://github.com/WordPress/gutenberg/pull/46112/commits/999b258ae546c63dfbd903e88ece6062a6e3c13a

Reply:
I'm not sure I follow. Perhaps you meant to comment on this other thread? https://github.com/WordPress/gutenberg/pull/46112/files#r1037282457

If so, how does it do nothing if this code stays in `lib/experimental`? The whole Gutenberg uses the class `WP_Theme_JSON_Resolver_Gutenberg` defined in this file.